object_manager.go

```diff
@@ -435,7 +472,7 @@ func (objMgr *ObjectManager) GetIpAddressFromHostRecord(host HostRecord) (string
 	return host.Ipv4Addrs[0].Ipv4Addr, err
 }
 
-func (objMgr *ObjectManager) UpdateHostRecordWithoutDNS(hostRref string, ipAddr string, macAddress string, vmID string) (string, error) {
+func (objMgr *ObjectManager) UpdateHostRecord(hostRref string, ipAddr string, macAddress string, vmID string) (string, error) {
```

Comment:
dont you need `vmName` just like above here?

Reply:
yes , i have missed vmName, thanks for pointing out